python/hsfs/core/vector_server.py

```diff
@@ -32,14 +33,22 @@
 
 
 class VectorServer:
-    def __init__(self, feature_store_id, features=[], training_dataset_version=None):
+    def __init__(
+        self,
+        feature_store_id,
+        features=[],
+        training_dataset_version=None,
+        serving_keys=None,
+    ):
         self._training_dataset_version = training_dataset_version
         self._features = features
+        self._feature_vector_col = [feat for feat in features if not feat.label]
```

Comment:
`fv.create_training_data()` any flavor of it throws error:

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-18-8940fa0ea0cb> in <module>
----> 1 fv1.create_training_data()

/srv/hops/anaconda/envs/theenv/lib/python3.10/site-packages/hsfs/feature_view.py in create_training_data(self, start_time, end_time, storage_connector, location, description, extra_filter, data_format, coalesce, seed, statistics_config, write_options, spine)
    872                 that was launched to create the training dataset.
    873         """
--> 874         td = training_dataset.TrainingDataset(
    875             name=self.name,
    876             version=None,

/srv/hops/anaconda/envs/theenv/lib/python3.10/site-packages/hsfs/training_dataset.py in __init__(self, name, version, data_format, featurestore_id, location, event_start_time, event_end_time, coalesce, description, storage_connector, splits, validation_size, test_size, train_start, train_end, validation_start, validation_end, test_start, test_end, seed, created, creator, features, statistics_config, featurestore_name, id, inode_id, training_dataset_type, from_query, querydto, label, transformation_functions, train_split, time_split_size, extra_filter)
    182             self._label = [feat.name.lower() for feat in self._features if feat.label]
    183             self._extra_filter = filter.Logic.from_response_json(extra_filter)
--> 184         self._vector_server = vector_server.VectorServer(
    185             featurestore_id, features=self._features
    186         )

/srv/hops/anaconda/envs/theenv/lib/python3.10/site-packages/hsfs/core/vector_server.py in __init__(self, feature_store_id, features, training_dataset_version, serving_keys)
     43         self._training_dataset_version = training_dataset_version
     44         self._features = features
---> 45         self._feature_vector_col = [feat for feat in features if not feat.label]
     46         self._prepared_statement_engine = None
     47         self._prepared_statements = None

TypeError: 'NoneType' object is not iterable